crates/ty_python_semantic/src/types.rs

```diff
@@ -7413,7 +7524,9 @@ impl<'db> VarianceInferable<'db> for Type<'db> {
             | Type::BoundSuper(_)
             | Type::TypeVar(_)
             | Type::TypedDict(_)
-            | Type::TypeAlias(_) => TypeVarVariance::Bivariant,
+            | Type::TypeAlias(_)
+            // REVIEWERS: Is this correct?
+            | Type::NewTypeInstance(_) => TypeVarVariance::Bivariant,
```

Comment:
I _think_ so? You can never have a "generic NewType" -- if it has a generic type as its underlying nominal class, that generic type _must_ have always been specialised. Therefore, it stands to reason that the typevar can never be relevant to this newtype, so must be bivariant with respect to this newtype.